src/common/components/MySecurity/Button/Button.style.ts

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components'
```

Comment:
As discussed in the group, we will use `TailwindCSS` instead of `styled-components` so please use `TailwindCSS` and remove `styled-components`. Please update for all files